config/vufind/PAIA.ini

```diff
@@ -18,6 +18,21 @@ baseUrl = ""
 ; same as for DAIA cache (see cacheLifetime setting in DAIA.ini).
 ;paiaCache = false
 
+; Specify what missing PAIA scopes should be reported to the user as account blocks.
+; For a comprehensive list of PAIA scopes refer to
+; http://gbv.github.io/paia/paia.html#access-tokens-and-scopes .
+;accountBlockNotificationsForMissingScopes[read_patron] = "no_read_patron_scope"
```

Comment:
I think all of these values need to have the `ILSMessages::` prefix added. I'd also add a comment explaining the format, like:

```
; Configuration format is accountBlockNotificationsForMissingScopes[scope] = message
; where scope is a PAIA scope name and message is the message to display to the user
; when that scope is missing. Messages will be translated.
```